src/server.ts

```diff
@@ -2,25 +2,86 @@ import express from "express";
 import dotenv from 'dotenv';
 import * as https from "https";
 import axios from "axios";
+import moment from "moment";
 
 dotenv.config();
+enum cameraType{
+    fhaz = "fhaz",
+    rhaz = "rhaz",
+    mast = "mast",
+    chamcam = "chamcam",
+    mahli = "mahli",
+    mardi = "mardi",
+    navcam = "navcam",
+    pancam = "pancam",
+    minites = "minites"
+}
+
+enum rover{
+    curiosity = "curiosity",
+    opportunity = "opportunity",
+    spirit = "spirit"
+}
 const app = express();
-const port = 8000;
 
 
 
-app.use(express.json());
+const port = 8000;
 
+app.use(express.json());
 const router = express.Router();
 // router.get('/test', (req: any, res: { send: (arg0: string) => any; }) => res.send());
+
 router.get('/rover', (req: any, res: { send: (arg0: string) => any; }) => {
     axios.get("https://api.nasa.gov/mars-photos/api/v1/rovers?api_key=" + process.env.API_KEY)
         .then((r: { data: any; }) => res.send(r.data))
         .catch(function(error){res.send(error)});
 });
 
+
+router.get('/rover/photos', (req:any, res: {send: (arg0:string)=> any}) => {
```

Comment:
What's the reasoning for using `arg0` here?

Reply:
IDE was being weird, changed now